lib/src/views/feed/components/post/post_actions.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:feed_sx/src/utils/constants/ui_constants.dart';
+import 'package:feed_sx/src/utils/constants/assets_constants.dart';
+import 'package:feed_sx/src/utils/constants/string_constants.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class PostActions extends StatelessWidget {
+  const PostActions({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Row(
+          children: [
+            kHorizontalPaddingSmall,
+            TextButton.icon(
+              onPressed: () {},
+              icon: SvgPicture.asset(kAssetLikeIcon),
+              label: const Text(
+                kStringLike,
+                style: TextStyle(fontSize: 14),
+              ),
+              style: ButtonStyle(
+                  textStyle: MaterialStateProperty.all(
+                      const TextStyle(fontSize: kButtonFontSize)),
+                  foregroundColor: MaterialStateProperty.all(kGrey2Color)),
+            ),
+            TextButton.icon(
+              onPressed: () {},
+              icon: SvgPicture.asset(kAssetCommentIcon),
+              label: const Text(
+                kStringAddComment,
+                style: TextStyle(fontSize: 14),
+              ),
+              style: ButtonStyle(
+                  textStyle: MaterialStateProperty.all(
+                      const TextStyle(fontSize: kButtonFontSize)),
+                  foregroundColor: MaterialStateProperty.all(kGrey2Color)),
+            )
+          ],
+        ),
+        const Spacer(),
+        Row(
+          children: [
+            IconButton(
+              onPressed: () {},
+              icon: SvgPicture.asset(kAssetBookmarkIcon),
```

Comment:
I know simple assets like these come builtin, we necessarily will not need to add the extra files.

Reply:
- Most of the Icons, like Heart, Share, Pin etc. are unavailable in Flutter's Icons Pack, so we should stick to using the designer's SVGs. It will be a bad practice to use Flutter's Icons Pack and SVGs together. Also, in future, if the client decides to change the icons according to their need, there is a high possibility that the inbuilt Icon pack might not fulfil their need.
- Also, let's not try to over-optimise the code right now, because, in case the SDK's size exceeds a certain limit, switching to the inbuilt icons pack won't be a headache.

Reply:
If you feel so, I still believe it would get the job done right now. I have used the same icons. And I think it would save the overhead of importing a whole package. Again, it's upto your personal choice.